### ISSUE TITLE
python_fx example problem

### ISSUE THREAD

**Issue by xinchen9** [intel/neural-compressor#46]
Run https://github.com/intel/neural-compressor/tree/master/examples/pytorch/nlp/huggingface_models/question-answering/optimization_pipeline/prune_once_for_all/fx script stage:
# for stage 1
python run_qa_no_trainer_pruneOFA.py --dataset_name squad \
      --model_name_or_path Intel/bert-base-uncased-sparse-90-unstructured-pruneofa \
      --teacher_model_name_or_path csarron/bert-base-uncased-squad-v1 \
      --do_prune --do_distillation --max_seq_length 384 --batch_size 12 \
      --learning_rate 1.5e-4 --do_eval --num_train_epochs 8 \
      --output_dir ./ --loss_weights 0 1 \
      --temperature 2 --seed 5143 --pad_to_max_length --run_teacher_logits

There are some errors during running:
![inc_quan_report](https://user-images.githubusercontent.com/93545498/153950017-f0977d71-3028-4ab1-9106-7f6f267057f8.png)

**Comment by XinyuYe-Intel** [comment]
Hi Xin, can you share your hardware and software environment? This error looks like the model and the input data is not on the same device, for example, the data in on gpu and the model is on the cpu.

**Comment by xinchen9** [comment]
Attachment is the conda env. The hardware:
GPU: Nvidia Quadro RTX8000.
CPU: Intel Xeon Platinum 8280

Xin

**Comment by XinyuYe-Intel** [comment]
Can you add below code on line 54 in run_qa_no_trainer_pruneOFA.py and try to run again, since we do not supported NV GPU in this example yet, thanks.
`os.environ["CUDA_VISIBLE_DEVICES"] = ""`

**Comment by chensuyue** [comment]
@xinchen9 have you try the patch? Could we close this issue?

**Comment by xinchen9** [comment]
Problem solved. with add os.environ["CUDA_VISIBLE_DEVICES"] = ""
